sklearn/utils/fixes.py

```diff
@@ -36,6 +36,9 @@ def _parse_version(version_string):
             version.append(x)
     return tuple(version)
 
+euler_gamma = getattr(np,
```

Comment:
`np` and `euler_gamma` could be put in a single line if you did this for pep8 line limit?